### DISCUSSION TITLE
how to turn off GUI

### DISCUSSION THREAD

**Discussion by fluidkocy** [InteractiveComputerGraphics/SPlisHSPlasH#274] [Q&A] [answered]
I want to turn off the GUI and extract only the data. For example, I would like to extract the force that a rigid body receives from a water particle. I also want to extract data of rigid bodies and water as vtk and bgeo files.

Please let me know how to do this.

**Comment by janbender** [comment, marked as answer]
You can use the command line option: --no-gui. However, you have to define an end time of the simulation in the scene file or by using the option "--stopAt". 

See also here: 
https://splishsplash.readthedocs.io/en/latest/getting_started.html

**Reply by fluidkocy** [reply]
Sorry for the low level question ....
How do I type a command line?

**Reply by janbender** [reply]
Just open a terminal or powershell window and start the simulator by calling "SPHSimulator".
